src/Microsoft.TestPlatform.ObjectModel/Client/Events/StopTestSessionCompleteEventArgs.cs

```diff
@@ -0,0 +1,49 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.VisualStudio.TestPlatform.ObjectModel.Client;
+
+using System;
+using System.Collections.Generic;
+using System.Runtime.Serialization;
+
+/// <summary>
+/// Event arguments used to notify the caller about the status of the test session.
+/// </summary>
+[DataContract]
+public class StopTestSessionCompleteEventArgs : EventArgs
+{
+    /// <summary>
+    /// Creates an instance of the current class.
+    /// </summary>
+    public StopTestSessionCompleteEventArgs()
+    { }
+
+    /// <summary>
+    /// Creates an instance of the current class.
+    /// </summary>
+    /// 
+    /// <param name="testSessionInfo">The test session info.</param>
+    public StopTestSessionCompleteEventArgs(TestSessionInfo testSessionInfo)
+    {
+        TestSessionInfo = testSessionInfo;
```

Comment:
null check? do we expect possible null here?

Reply:
No need to null check, if the `testSessionInfo` is null the `TestSessionInfo` property will be null too as it is by default.